core/src/main/java/org/pac4j/jax/rs/filters/CallbackFilter.java

```diff
@@ -26,10 +26,6 @@ public class CallbackFilter extends AbstractFilter {
 
     private String defaultUrl;
 
-    private Boolean saveInSession;
```

Comment:
Similarly, I didn't see `saveInSession` in [CallbackLogic](https://github.com/pac4j/pac4j/blob/pac4j-parent-5.7.1/pac4j-core/src/main/java/org/pac4j/core/engine/CallbackLogic.java#L29-L30)